controllers/specialresource.go

```diff
@@ -57,7 +58,7 @@ func SpecialResourcesReconcile(ctx context.Context, r *SpecialResourceReconciler
 		}
 		parent, err := r.Storage.CheckConfigMapEntry(ctx, req.Name, obj)
 		if err != nil {
-			r.StatusUpdater.UpdateWithState(ctx, &r.parent, fmt.Sprintf("%v", err))
+			log.Error(err, "")
```

Comment:
I think this should have an error message.

Reply:
You're right, added